test/functional/fixtures/fail-test.js

```diff
@@ -0,0 +1,32 @@
+var runTest = require('../test-runner.js').runTests;
+var expect  = require('chai').expect;
+
+
+describe('fail test', function () {
```

Comment:
Maybe I was unclear before. We are preparing this PR to pushing in the main repo. We should not add fake tests. We should add real tests to demonstrate how to use this. So we can get two `act.click` tests for this purposes. One tests with the `skip` directive, the second - the test that should fail. I gave link for them in one of the previous comments